common/lib/capa/capa/tests/test_shuffle.py

```diff
@@ -194,7 +194,10 @@ def test_shuffle_fixed_both_ends(self):
         """)
         problem = new_loncapa_problem(xml_str, seed=0)
         the_html = problem.get_html()
-        self.assertRegexpMatches(the_html, r"<div>.*\[.*'Alpha'.*'Beta'.*'B'.*'A'.*'C'.*'D'.*'Psi'.*'Omega'.*\].*</div>")
+        self.assertRegexpMatches(
```

Comment:
This PR somehow got over the pylint error threshold, even though there are no errors on the lines I changed. I have picked a random error to fix to get back to the threshold (and this is in a separate commit).